sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1324,6 +1359,11 @@ def task_worker(self):
                                self.post_port_active_apsel_to_db(api, lport, host_lanes_mask, reset_apsel=True)
                            self.update_port_transceiver_status_table_sw_cmis_state(lport, CMIS_STATE_READY)
                            continue
+                        elif self.is_decommission_required(lport, api):
+                            self.port_dict[lport]['is_decomm_required'] = True
+                            api.decommission_all_datapaths(False)
```

Comment:
@AnoopKamath are you proposing to change the API? It does not take any argument True/False?

Reply:
@prgeor : yes, here is the PR: https://github.com/sonic-net/sonic-platform-common/pull/565